config/settings/production.py

```diff
@@ -260,15 +261,19 @@
 if HEROKU_TOKEN and HEROKU_APP_NAME:
     METACI_WORKER_AUTOSCALER = "metaci.build.autoscaling.HerokuAutoscaler"
 
-DEFAULT_AUTOSCALERS = {
-    "mrbelvedereci": {
-        "app_name": "mrbelvedereci",
-        "worker_type": "worker",
-        "queues": ["default", "medium", "high"],
-    }
-}
 # Autoscalers are defined per METACI_APP
-AUTOSCALERS = json.loads(env("AUTOSCALERS", default=DEFAULT_AUTOSCALERS))
+AUTOSCALERS = json.loads(env("AUTOSCALERS", default={}))
+
+if not AUTOSCALERS:
+    # backwards compatability
+    HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
+    AUTOSCALERS = {
+        HEROKU_APP_NAME: {
+            "app_name": HEROKU_APP_NAME,
+            "worker_type": "worker",
+            "queues": ["default", "medium", "high"],
+        }
+    }
```

Comment:
@Br4nd0R If HEROKU_APP_NAME is None, we should set AUTOSCALERS to an empty dict. (i.e. don't autoscale)